apps/e2e/ag-grid-storybook-e2e/src/integration/data-entry-grid.component.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('ag-grid-storybook', () => {
+  E2eVariations.forEachTheme((theme) => {
+    describe(`in ${theme} theme`, () => {
+      it('should edit date and lookup fields', () => {
```

Comment:
This test captures a date field with and without the calendar open, and captures a lookup field without showing the results.